api/index.py

```
@@ -11,10 +11,18 @@
 project_root = Path(__file__).parent.parent
 sys.path.insert(0, str(project_root))
 
+# Load environment variables early for Vercel
+import os
```

Ruff (0.11.9)

Module level import not at top of file

(E402)